client/app/models/bbl.js

```diff
@@ -1,5 +1,32 @@
 import Model, { attr, belongsTo } from '@ember-data/model';
 
+export const PROJECT_BBL_BOROUGHS = {
```

Comment:
It's not super clear _why_ it's necessary that you're mapping these codes -- are you sending these codes to a CRM field, e.g. `dcp_borough`? Would love a comment here explaining why it's necessary

Reply:
will do! it's necessary because zap search provides bbls as 1-5-4 numerics, and we need to map the first numeric to a particular CRM code.